cli/cook/subcommands/ssh.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import os
+from operator import itemgetter
+from urllib.parse import urlparse
+
+from cook import colors, http
+from cook.subcommands.show import query, print_no_data
```

Comment:
Should we factor out functions shared by subcommands so that they don't depend on each other?  `query` and `print_no_data` seem general enough to not be specifically in the `show` command.

Reply:
Done (created `querying.py`)